parser/Builder.cc

```diff
@@ -28,6 +28,8 @@ using std::type_info;
 using std::unique_ptr;
 using std::vector;
 
+extern const char *dclassStrings[];
+
```

Comment:
Why is this needed?

Reply:
good catch, this was the result of a bad `git add -p`, it's not needed.
```suggestion
```